### ISSUE TITLE
HOW TRANSFER REPEAT POKEMON WITH BOT 

### ISSUE THREAD

**Issue by danielocampbells** [PokemonGoF/PokemonGo-Bot-Desktop#244]
In config.json is possibly set transfer repeat pokemon ? 

_Issue without proper information won't be answered_
### Information
- OS
- Git commit ( to get git commit run `git log -1 --format="%H"`)
- App's logs

**Comment by mrincredible2017** [comment]
Yes just look in the config.json it's easy to figure out, but do you mean transfer all duplicates? With no regard for CP/IV?

**Comment by danielocampbells** [comment]
yeah i want to transfer duplicates, for example, i have a lot of pidgeys, and i want keep only maybe 2 best and the rest transfer for candy, its possibly ? get the code?

**Comment by Pwntastickevin** [comment]
Great Request ^

**Comment by KDA809** [comment]
this can be done in the config.json read through documentation 👍 it will help and if ur new to it all Sublime editor is great tool to edit file an save

**Comment by IrishReign** [comment]
@iiCe89 How would I do something like exchange all duplicates under 700?

**Comment by KDA809** [comment]
@IrishReign Keep the strongest pokemon configuration (dev branch)

[back to top]

You can set "release": {"Pidgey": {"keep_best_cp": 1}} or "release": {"any": {"keep_best_iv": 1}}.

In that case after each capture bot will check that do you have a new Pokémon or not.

If you don't have it, it will keep it (no matter was it strong or weak Pokémon).

If you already have it, it will keep a stronger version and will transfer the a weaker one.

`"release": {"any": {"keep_best_cp": 2}}, "release": {"any": {"keep_best_cp": 10}} - can be any number. In the latter case for every pokemon type bot will keep no more that 10 best CP pokemon.
`

If you wish to limit your pokemon bag as a whole, not type-wise, use all: "release": {"all": {"keep_best_cp": 200}}. In this case bot looks for 200 best CP pokemon in bag independently of their type. For example, if you have 150 Snorlax with 1500 CP and 100 Pidgeys with CP 100, bot will keep 150 Snorlax and 50 Pidgeys for a total of 200 best CP pokemon.

**Comment by KDA809** [comment]
you can set config.json to release pokemon 700 & below if thats what u mean

**Comment by IrishReign** [comment]
@iiCe89 Right now I have it set to discard below cp600 but I want to keep it like that. How would I make it so that it would discard all duplicates under 900cp?

**Comment by IrishReign** [comment]
@iiCe89 How can I set it to discad all duplacates under 800 cp, exept to keep everything 500 cp and above?
